README.md

```diff
@@ -1,45 +1,99 @@
+# Sherlock Platform
+
+## About
+
+This is the customized IntelliJ platform for Sherlock Project - an IntelliJ-based implementation of the new GPU Profiler tool. This is based
```

Comment:
Can you please delete the non-public information:  `an IntelliJ-based implementation of the ..." part?